### PR TITLE
Updated load_template function calling and read_coordinates ()

### LINKED ISSUE #40: Update the load_template function call in both write_dirs_files_for_gamess_qcxms.py and extract_coords_into_inp_xyz.py.
The error below appeared when running the scripts `write_dirs_files_for_gamess_qcxms.py` and `extract_coords_into_inp_xyz.py` on the HPC. 

Input file used: 
[scores_lower_than_200_mol_per_class_name_smiles.zip](https://github.com/user-attachments/files/17364786/scores_lower_than_200_mol_per_class_name_smiles.zip)

   ` qcxms_template = load_template("templates", "qcxms_input_template.in")
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: load_template() takes 1 positional argument but 2 were given`

Hint: Update the function call to load_template("qcxms_input_template.in")

### LINKED ISSUE #42: Function read_coordinates() in extract_coords_into_inp_xyz.py fails
It fails when using the following input file: 
[scores_lower_than_200_mol_per_class_name_smiles.zip](https://github.com/user-attachments/files/17364989/scores_lower_than_200_mol_per_class_name_smiles.zip)

It works with the older version - written below:

``` 
def read_coordinates(n_atoms, gamess_output, line_number):
    start, end = compute_start_end_indices(n_atoms, line_number, 4)
    data = []
    for i in range(start, end):
    data.append(gamess_output[i].split())
    atomic_number = []
    element_symbol = []
    x_coord = []
    y_coord = []
    z_coord = []
    for line in data:
        element_symbol.append(line[0])
        atomic_number.append(line[1])
        x_coord.append(line[2])
        y_coord.append(line[3])
        z_coord.append(line[4])
    return element_symbol,x_coord,y_coord,z_coord

### PR DESCRIPTION
**Pull request by wverastegui**
This PR updated the calling format of the function load_template() in the scripts `extract_coords_into_inp_xyz.py` and `write_dirs_files_for_gamess_qcxms.py`.

The error in the read_coordinates() function in extract_coords_into_inp_xyz.py has been resolved by reverting to the previous version.

closes https://github.com/RECETOX/ei_spectra_predictions/issues/42
closes https://github.com/RECETOX/ei_spectra_predictions/issues/40

### CODE REVIEW

File: codes/write_dirs_files_for_gamess_qcxms.py [hunk 1]
```diff
@@ -277,13 +277,13 @@ def write_gamess_input(multiplicity: int, mol: object, molname: str, mol_input_p
             write_gamess_input(multiplicity, mol, molname, mol_input_path)
 
         spectrum_input_path = spectra_dir / ("qcxms" + ".in")
-        qcxms_template = load_template("templates", "qcxms_input_template.in")
+        qcxms_template = load_template("qcxms_input_template.in")
         if not Path(spectrum_input_path).exists():
-            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit", "ieeatm"])
+            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit"])
```
Comment (hechth) [inline codes/write_dirs_files_for_gamess_qcxms.py]: Why are we not reading the 'ieeatm' anymore?

File: codes/write_dirs_files_for_gamess_qcxms.py [hunk 2]
```diff
@@ -277,13 +277,13 @@ def write_gamess_input(multiplicity: int, mol: object, molname: str, mol_input_p
...
             write_from_template(parameters=qcxms_params, template=qcxms_template, file=spectrum_input_path)
 
         mol_pbs_path = mol_dir / (inchikey + ".pbs")
-        pbs_template = load_template("templates", "optimization_gamess_template.pbs")
+        pbs_template = load_template("optimization_gamess_template.pbs")
```
Comment (hechth) [inline codes/write_dirs_files_for_gamess_qcxms.py]: You can re-use the PBS_TEMPLATE variable here

File: codes/write_dirs_files_for_gamess_qcxms.py (hunk 1 above)
Comment (wverastegui) [reply codes/write_dirs_files_for_gamess_qcxms.py]: It was a mistake. Fixed [b6d895c](https://github.com/RECETOX/ei_spectra_predictions/pull/43/commits/b6d895c20db06f75e49f5f55bb4c9584f15267b6)

File: codes/write_dirs_files_for_gamess_qcxms.py (hunk 2 above)
Comment (wverastegui) [reply codes/write_dirs_files_for_gamess_qcxms.py]: I use one variable now: `template` [2ffd3ef](https://github.com/RECETOX/ei_spectra_predictions/pull/43/commits/2ffd3ef950cbfe0514a2c25a138e8da90901d371)

Comment (wverastegui) [reply codes/write_dirs_files_for_gamess_qcxms.py]: @hechth, If you agree. We can merge this PR?

Comment (hechth) [reply codes/write_dirs_files_for_gamess_qcxms.py]: this is not what I meant - now you are overwriting the variable, not sure if this will work.

**Pull request merged**